_includes/tutorials/transforming/kafka/code/configuration/test.properties

```diff
@@ -1,6 +1,6 @@
-confluent.version=5.3.0
+confluent.version=6.0.0
 bootstrap.servers=localhost:29092
-schema.registry.url=http://localhost:8081
+schema.registry.url=mock://localhost:8081
```

Comment:
This change is necessary to work around not using SR testcontainer and use the mock SR instead